controllers/ratelimit_policy_status_updater.go

```diff
@@ -113,6 +113,10 @@ func (r *RateLimitPolicyStatusUpdater) UpdateStatus(ctx context.Context, _ []con
 }
 
 func (r *RateLimitPolicyStatusUpdater) enforcedCondition(policy *kuadrantv1.RateLimitPolicy, topology *machinery.Topology, state *sync.Map) *metav1.Condition {
+	kObj := GetKuadrantFromTopology(topology)
+	if kObj == nil {
+		return kuadrant.EnforcedCondition(policy, kuadrant.NewErrSystemResource("kuadrant"), false)
+	}
```

Comment:
Should this also be done for `AuthPolicy` ? The current condition for `AuthPolicy` without Kuadrant installed is:

```yaml
  status:
    conditions:
    - lastTransitionTime: "2024-11-13T08:50:35Z"
      message: AuthPolicy has been accepted
      reason: Accepted
      status: "True"
      type: Accepted
    - lastTransitionTime: "2024-11-13T08:50:54Z"
      message: 'AuthPolicy has encountered some issues: missing auth effective policies
        stored in the reconciliation state'
      reason: Unknown
      status: "False"
      type: Enforced
```

Reply:
Yes this should have being done.